client/src/components/style/dropdownfilter.css

```diff
@@ -0,0 +1,74 @@
+.form {
+  width: 18rem;
+  margin: 0.5rem 0;
+  padding: 0;
+  display: block;
+}
+
+.formselect {
+  font-size: 0.875rem;
+  box-sizing: border-box;
+  width: 100%;
+  height: 30px;
+  /* margin: 4px 0; */
```

Comment:
why comment?

Reply:
I was playing around with the margins and wanted to keep them in there in case I wanted to use later.